sdk/keyvault/keyvault-keys/.prettierignore

```diff
@@ -1 +1,2 @@
-src/core
\ No newline at end of file
+/src/core/*
```

Comment:
Why the root slash? Does that still always resolve to the root? The old line didn't have it.

Reply:
Without the root hash, I can't re-include the challengeBasedAuth file that is in the code folder :/ I will clean this up once we move this to a common folder.

Reply:
huh, why are we ignoring these files anyway?

Reply:
Here's the common folder PR that I'm proposing: https://github.com/Azure/azure-sdk-for-js/pull/8866

Reply:
@xirzec the files in the src/core folder should only come from the swagger tool we use... except for the challenge based authentication code! which we should move out of there.

Reply:
ohh... can we rename the folder to 'generated' at some point to make this clear?

Reply:
Here's an issue: https://github.com/Azure/azure-sdk-for-js/issues/9069